packages/core/contracts/optimistic-asserter/implementation/OptimisticAsserter.sol

```diff
@@ -431,22 +429,22 @@ contract OptimisticAsserter is OptimisticAsserterInterface, Lockable, Ownable, M
         return _getOracle(assertionId).getPrice(identifier, time, _stampAssertion(assertionId));
     }
 
-    function _getEscalationManager(bytes32 assertionId) internal view returns (EscalationManagerInterface) {
-        return EscalationManagerInterface(assertions[assertionId].escalationManagerSettings.escalationManager);
+    function _getEscalationManager(bytes32 assertionId) internal view returns (address) {
```

Comment:
I'm not sure if I view this as an improvement. we are using this thing as an `EscalationManagerInterface` in almost all cases and this means we need to do additional type casting. we could also simplfy this bit of code on line 443 and 449
```
        return EscalationManagerInterface(em).getAssertionPolicy(assertionId);

``` to not have to re-cast this to `EscalationManagerInterface` if we returned `EscalationManagerInterface` rather than an address.

Reply:
If `_getEscalationManager` returned `EscalationManagerInterface` then each of consuming methods would still need to do type conversion when checking non-zero address